package/MDAnalysis/analysis/pca.py

```diff
@@ -123,32 +132,92 @@ class PCA(AnalysisBase):
     principal components ordering the atom coordinate data by decreasing
     variance will be available for analysis. As an example:
 
-        >>> pca = PCA(universe, select='backbone').run()
-        >>> pca_space =  pca.transform(universe.select_atoms('backbone'), 3)
+    .. code-block:: python
```

Comment:
technically (=NumpyDoc) speaking, the synopsis should not be a code example. However, we're not fixing docs right not so I am more than happy to just have some stylistic changes & sort out docs later.

Reply:
Starting points for docs is https://userguide.mdanalysis.org/stable/contributing_code.html#working-with-the-code-documentation

Reply:
Thanks. I'm would especially like to make consistent docs for classes and will open an issue when 2.0 is out.